CactusTutorial.ipynb

```diff
@@ -535,11 +545,117 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# start simulation, watch log output\n",
-    "!./simfactory/bin/sim create-run tov_ET \\\n",
+    "%%bash\n",
+    "# start simulation\n",
+    "./simfactory/bin/sim create-submit tov_ET \\\n",
     "  --parfile=par/tov_ET.par --procs=2 --num-threads=1 --walltime=0:20:0"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `submit` command submitted the simulation to the queueing system to run in the background asking for a maximum runtime of 20 minutes. On your laptop it will start right away, on a cluster the queuing system will wait until a sufficient number of nodes is able to start your simulation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can check the status of the simulation with the command below. You can run this command repeatedly until the job shows <pre>[ACTIVE (FINISHED)...</pre> as it's state. Prior to that, it may show up as QUEUED or RUNNING."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "bash: line 1: simfactory/bin/sim: No such file or directory\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "simfactory/bin/sim list-simulations tov)ET"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    " To watch its output use the `show-output` command of simfactory. **Interrupt the kernel** (or press `CTRL-C` if copying & pasting these commands to a terminal) once you wish to stop watching or once the simulation has ended."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "# watch log output, following along as new output is produced\n",
+    "./simfactory/bin/sim show-output --follow tov_ET"
```

Comment:
This cell does not exit. It has to be stopped with interrupt. If we used %%jbash here, it might work better.

Reply:
The cell does not exit b/c the `--follow` option make `show-output` never exit. There is bolded text above that reads "Interrupt the kernel (or press CTRL-C if copying & pasting these commands to a terminal) once you wish to stop watching or once the simulation has ended.". I have added more bolded text just above the next cell (which will seem to hang) stating "Remember that you have to interrupt the kernel to stop show-output and be able to execute the cells below." as I was also caught be not realizing one has to interrupt even after the simulation finished.